src/CocktailDetails/CocktailDetails.tsx

```diff
@@ -1,66 +1,125 @@
-import React, { useState, useEffect } from 'react';
-import { getCocktailDetails } from '../apiCalls';
-import { Cocktail } from '../Definitions/RandomCocktail'
-import './CocktailDetails.scss';
+import React, { useState, useEffect } from "react";
+import { getCocktailDetails } from "../apiCalls";
+import { Cocktail } from "../Definitions/RandomCocktail";
+import "./CocktailDetails.scss";
+
 export interface CocktailDetailsProps {
-	id: string;
+  id: string;
+  toggleFavorites: (id: string) => any;
```

Comment:
Added any type since toggleFavorites doesn't have a return value.

Reply:
I think the return type is supposed to be void for functions not returning anything. (https://stackoverflow.com/questions/49202813/typescript-return-type-void)

Reply:
Weren't we having an error when we attempted to pass it when we had void - I seem to recall that we had void, and changed it to any.

Reply:
From what I gathered between what my mentor has told me and reading it the SO post, this is how I understand it:
- It's fine to use any or void when a function doesn't return anything, but that she personally uses any
- From the SO post, it's better to type it with void vs not typing it all